test/unit/pcv/NamedStaticPCVDepositWrapper.test.ts

```diff
@@ -0,0 +1,302 @@
+import { getCore, getAddresses, expectRevert, expectEvent, getImpersonatedSigner } from '@test/helpers';
+import { expect } from 'chai';
+import hre, { ethers } from 'hardhat';
+import { Signer } from 'ethers';
+import { Core, Fei, NamedStaticPCVDepositWrapper } from '@custom-types/contracts';
+const toBN = ethers.BigNumber.from;
+
+describe('NamedStaticPCVDepositWrapper', function () {
+  const impersonatedSigners: { [key: string]: Signer } = {};
+
+  let underlyingTokenAmount: string;
+  let governorAddress: string;
+  let newDepositName: string;
+  let balance: string;
+  let feiBalance: string;
+  let core: Core;
+  let deposit: NamedStaticPCVDepositWrapper;
+
+  before(async () => {
+    const addresses = await getAddresses();
+
+    // add any addresses you want to impersonate here
+    const impersonatedAddresses = [addresses.governorAddress];
+
+    for (const address of impersonatedAddresses) {
+      impersonatedSigners[address] = await getImpersonatedSigner(address);
+    }
+  });
+
+  beforeEach(async function () {
+    ({ governorAddress } = await getAddresses());
+
+    newDepositName = 'Visor Finance Deposit';
+    balance = '2000';
+    feiBalance = '1000';
+    underlyingTokenAmount = '100000';
+    core = await getCore();
+    deposit = await (
+      await ethers.getContractFactory('NamedStaticPCVDepositWrapper')
+    ).deploy(core.address, [
+      {
+        depositName: newDepositName,
+        usdAmount: balance, /// USD equivalent in this deposit, not including FEI value
+        feiAmount: feiBalance, /// amount of FEI in this deposit
+        underlyingTokenAmount: underlyingTokenAmount, /// amount of underlying token in this deposit
+        underlyingToken: await core.fei()
+      }
+    ]);
+  });
+
+  describe('init', function () {
+    it('reported in USD', async function () {
+      expect(await deposit.balanceReportedIn()).to.be.equal('0x1111111111111111111111111111111111111111');
+    });
+
+    it('depositName', async function () {
```

Comment:
depositName --> depositName is correct

Reply:
Not sure why we need to change the names on these init functions as this has been convention to do function name for init.